torch/autograd/__init__.py

```diff
@@ -50,7 +51,15 @@ def _make_grads(outputs, grads):
     return tuple(new_grads)
 
 
-def backward(tensors, grad_tensors=None, retain_graph=None, create_graph=False, grad_variables=None):
+_TensorOrTensors = Union[torch.Tensor, Sequence[torch.Tensor]]
+
+
+def backward(
+    tensors: _TensorOrTensors,
+    grad_tensors: Optional[_TensorOrTensors] = None,
+    retain_graph: Optional[bool] = None,
+    create_graph: bool = None
```

Comment:
You lost `grad_variables` here, it's not documented but still used in the function body.